ui/helpers/constants/design-system.ts

```diff
@@ -22,12 +22,10 @@ export enum Color {
   primaryAlternative = 'primary-alternative',
   primaryMuted = 'primary-muted',
   primaryInverse = 'primary-inverse',
-  primaryDisabled = 'primary-disabled',
   errorDefault = 'error-default',
   errorAlternative = 'error-alternative',
   errorMuted = 'error-muted',
   errorInverse = 'error-inverse',
-  errorDisabled = 'error-disabled',
```

Comment:
no instances of `errorDisabled` used in code base

![Screenshot 2024-05-31 at 4 29 23 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/3f374249-39b7-4f0c-bbcb-834864b4849f)